Libraries/LibGUI/GFilePicker.h

```diff
@@ -7,9 +8,20 @@ class GLabel;
 
 class GFilePicker final : public GDialog {
 public:
-    GFilePicker(const StringView& path = "/", CObject* parent = nullptr);
+    enum class Mode {
+        Open,
+        Save
+    };
+
+    static Optional<String> get_open_filepath();
+    static Optional<String> get_save_filepath();
+
+    GFilePicker(Mode type = Mode::Open, const StringView& path = "/", CObject* parent = nullptr);
     virtual ~GFilePicker() override;
 
+    bool create_file(const StringView& path);
+    bool file_exists(const StringView& path);
```

Comment:
This function does not belong in the `GFilePicker` API.
Instead, it can just be a `static` local function in `GFilePicker.cpp`